go/scanner/scanner.go

```diff
@@ -150,7 +150,7 @@ func DefaultScannerOptions() *ScannerOptions {
 // Scanner is a tool to scan all the entries in a CT Log.
 type Scanner struct {
 	// Client used to talk to the CT log instance
-	logClient *client.LogClient
+	logClient client.LogClient
```

Comment:
Yeah, interfaces are a pointer type, I understand (when you call something that takes an `interface{}`, for example, you have to do `&foo`), but I wasn't sure if `*LogClient` would be flagged as an error by the compiler, or whether it's allowed, but `go vet` would frown a bit, or if you just searched around by hand to fix those?